src/script/auth/page/SingleSignOnForm.tsx

```diff
@@ -187,11 +187,11 @@ class _SingleSignOnForm extends React.PureComponent<Props & ConnectedProps & Dis
           }
           default: {
             this.setState({ssoError: error});
-            const isValidationError = Object.values(ValidationError.ERROR).some(errorType =>
-              error.label.endsWith(errorType)
+            const isValidationError = Object.values(ValidationError.ERROR).some(
+              errorType => error.label && error.label.endsWith(errorType)
             );
             if (!isValidationError) {
-              throw error;
+              console.warn('SSO authentication error', JSON.stringify(Object.entries(error)));
```

Comment:
```suggestion
              console.warn('SSO authentication error', JSON.stringify(Object.entries(error)), error);
```

Reply:
There's no value in doing that because the error doesn't contain any useful data in this form.

Reply:
![image](https://user-images.githubusercontent.com/1138344/57237473-8fcb1f80-7027-11e9-9750-78a960582d11.png)

Reply:
But you will get the stack trace.